### DISCUSSION TITLE
How to solve the error 'Error: Cannot read properties of null (reading 'protocol')'?

### DISCUSSION THREAD

**Discussion by lxy071130** [gitalk/gitalk#547] [Q&A]
I use gitalk as what the instruction said and I got this error.I have no idea why this is happening and waiting for help.

**Comment by marcushaghes** [comment]
My grades were really low, had a GPA of 1.28, and was almost asked to leave the college if fail the next exam coming, I was really scared at that point but then I came across zeusmasterhack. I spoke with him and I was told zeusmasterhack would get my grades changed from F to A or my exam questions and answer. Never believed him at first because I felt it wasn’t possible but on second thought I had no choice but to give him a trial. I paid for his tools and in a few hours he sent me the questions and answers to my courses. On the day of my exams, the questions were exactly the same as what they sent to me. I was so happy and pleased By the help. zeusmasterhack contact him on his mail zeusmasterhack@gmail.com  did the impossible, never believed my college website could be hacked
